feature/dsql/token/auth_token_generator.go

```diff
@@ -0,0 +1,121 @@
+package token
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	v4 "github.com/aws/aws-sdk-go-v2/aws/signer/v4"
+	"github.com/aws/aws-sdk-go-v2/internal/sdk"
+)
+
+const (
+	rdsClusterTokenID = "dsql"
```

Comment:
This is actually vendor code
```suggestion
	vendorCode = "dsql"
```